app/views/events/index.html.haml

```diff
@@ -2,18 +2,19 @@
 = content_for :page_header do
   %h1= t('.title', :default => model_class.model_name.human(count: 2))
 
-.navbar
-  .navbar-inner
-    = link_to t('.new', :default => t("helpers.links.new")), new_resource_path, :class => 'btn btn-primary', :data => { :confirm => t(:new_event_warning) }
+%nav.navbar.navbar-default
+  .container-fluid
```

Comment:
@nikhilgupta1211 The .container-fluid causes the 'New' button to be aligned differently (some pixels further to the right). Also the form differs quite a bit from what it look like before.

Latter one seems to be caused by replacing 'input-small' with 'form-control'. The [bootstrap docs](http://getbootstrap.com/migration/#dropped) recommend to "Use .form-control and the [grid system](http://getbootstrap.com/css/#grid) instead.".
You might have to apply this here.